src/renderer/components/Login.vue

```diff
@@ -1,19 +1,110 @@
 <template>
-  <el-card>
-    <img id="logo" src="~@/assets/logo.svg" alt="Iroha">
-    <span>Login page, Welcome!</span>
+  <el-card class="login-form-container">
+    <img id="logo" class="logo" src="~@/assets/logo.svg" alt="Iroha">
+
+    <el-form ref="form" :model="form" :rules="rules" label-position="top">
+      <el-form-item label="username:" prop="username">
+        <el-input name="username" v-model="form.username"></el-input>
+      </el-form-item>
+
+      <el-form-item label="private key:" prop="privateKey">
+        <el-row type="flex" justify="space-between">
+          <el-col :span="17">
+            <el-input name="privateKey" v-model="form.privateKey"></el-input>
+          </el-col>
+
+          <el-col :span="6">
+            <el-upload
+              action=""
+              :auto-upload="false"
+              :show-file-list="false"
+              :on-change="onFileChosen"
+            >
+              <el-button type="primary">choose</el-button>
```

Comment:
let's change type of this button from primary to some other, maybe button with outlines, so it doesn't distruct user from "login" button. Also, maybe let's make this button with icon inside, not text.

Reply:
Let's remove text and put icon only.